libBuild.sh

```diff
@@ -239,4 +239,71 @@ function publish_layer {
       --principal "*" \
       --region "$region"
     echo "Public permissions set for ${runtime_name} layer version ${layer_version} in region ${region}"
+
+    # Creating layer as a docker image and publishing it in ECR 
+    if [ "$region" = "us-east-1" ]; then
+        publish_docker_ecr $layer_archive $region $runtime_name $arch $layer_name $layer_version
+    fi
+
+
+}
+
+function publish_docker_ecr {
+    layer_archive=$1
+    region=$2
```

Comment:
we are not using the region later as we are hardcoding it to us-east-1. Might be we can remove it